benches/benches/bevy_ecs/change_detection.rs

```diff
@@ -86,7 +86,7 @@ fn generic_bench<P: Copy>(
 
 fn all_added_detection_generic<T: Component + Default>(group: &mut BenchGroup, entity_count: u32) {
     group.bench_function(
-        format!("{}_entities_{}", entity_count, std::any::type_name::<T>()),
+        format!("{}_entities_{}", entity_count, core::any::type_name::<T>()),
```

Comment:
Most changes look like this, where some inline call to a `std` item needs to be replaced with the `core`/`alloc` alternative. As `std` re-exports `alloc` and `core`, this is a non-breaking change even in public facing signatures.